bitcoin/base58.c

```diff
@@ -27,9 +27,10 @@ static char *to_base58(const tal_t *ctx, u8 version,
 	if (wally_base58_from_bytes((const unsigned char *) buf, total_length, BASE58_FLAG_CHECKSUM, &out) != WALLY_OK) {
 		return NULL;
 	}else{
-		char *res;
+		char *res = NULL;
```

Comment:
Rusty won't like this since it interferes with `valgrind`s ability to tell us about accesses to uninitialized fields.

Reply:
@cdecker fair enough! updated in https://github.com/ElementsProject/lightning/pull/2594/commits/dc413ba6f043254115e7b8a84f4e1aed1ed5bcaa - let me know when you want me to rebase or squash